device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2

```diff
@@ -1,4 +1,6 @@
 {% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %}
+{% set different_dscp_to_tc_map = true %}
```

Comment:
need same set of changes for C32 sku as well

Reply:
The [Arista-7050CX3-32S-C32](https://github.com/sonic-net/sonic-buildimage/tree/202012/device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32)/[BALANCED](https://github.com/sonic-net/sonic-buildimage/tree/202012/device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/BALANCED)/qos.json.j2 is linked to [Arista-7050CX3-32S-D48C8/BALANCED/qos.json.j2](https://github.com/sonic-net/sonic-buildimage/pull/12792/files/0422dc61d894592045f7a886a70c450c74791b1c#diff-2fe712257b431ce432d1903123a08e1cf0dc304597369e0a44acbb60c196888a)